test/acceptance/news-mp3-redirects.hurl

```diff
@@ -0,0 +1,251 @@
+GET {{proto}}://{{host}}/uploads/podcast/news-2022-06-27/the-changelog-news-2022-06-27.mp3
+Host: {{assets_host}}
+HTTP 308 # expect permanent redirect response
+[Asserts]
+header "Location" == "https://{{assets_host}}/uploads/news/1/changelog-news-1.mp3"
```

Comment:
I recommend adding some of the following:

- `http://{{old-news-url}}` redirects to `https://{{new-news-url}}` -- change protocol
- `https://www.{{old-news-url}}` redirects to `https://{{new-news-url}}` -- remove subdomain
- `http://www.{{old-news-url}}` redirects to `https://{{new-news-url}}` -- change protocol and remove subdomain

Reply:
OK, will add more tests for http -> https redirects, not just for the root domain.